src/mutate-errors-to-warnings.ts

```diff
@@ -0,0 +1,32 @@
+import { generateFileName, _buildTodoDatum } from '@ember-template-lint/todo-utils';
+import { TodoData } from '@ember-template-lint/todo-utils/lib/types';
```

Comment:
Let's make a note that we need to export the types from that package, as 'digging in' to a private module is brittle and should be avoided.

Reply:
I added a `TODO` comment to note this.

Reply:
Also created https://github.com/ember-template-lint/ember-template-lint-todo-utils/issues/18